blockchain-tests-skip.yml

```diff
@@ -304,6 +304,7 @@ filename:
     - CREATE_EContractCreateNEContractInInit_Tr.json # ef-tests #340
     - CREATE_empty000CreateinInitCode_Transaction.json # ef-tests #340
     - CREATE_FirstByte_loop.json # ef-tests #340
+    - CreateTransactionHighNonce.json # ef-tests #471
```

Comment:
why are we adding this one again?

Reply:
This is the one that causes blockifier to crash, due to increasing u64::max nonce by one :/